test/e2e/service.go

```diff
@@ -840,6 +845,33 @@ var _ = Describe("Services", func() {
 		for _, svc := range svcs {
 			namespace := svc.Namespace
 			lbip := svc.Spec.LoadBalancerIP
+			defer func() {
```

Comment:
looking more closely, this seems to be a copy-paste of similar code in the other test? maybe just factor it out into a helper function and call that from both places?